.github/workflows/publish.yaml

```diff
@@ -0,0 +1,34 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - '0.*'
+      - '1.*'
+
+jobs:
```

Comment:
Do you want a `needs` that has some of the jobs from `ci.yaml`? We have it in ops, and I guess it adds extra reliability, but it also does seem unnecessary because (assuming no-one with the required permissions overrides) those ought to have been ok when merging to main and shouldn't break afterwards.

I haven't thought about this in depth, but I think my preference is what you have here and potentially doing that in ops as well.

Reply:
Yeah, I thought about that too, and I think because we run all these on every PR (and merge into main), it's not necessary to repeat that here.